src/main/java/com/sarahhand/fractals/model/ColorScheme.java

```diff
@@ -25,7 +23,14 @@
 })
 public interface ColorScheme{
 	
+	@SuppressWarnings("javadoc")
```

Comment:
It's good programming practice to always provide Javadoc for interface methods since these define the "contract" between implementing classes and the client class that uses the interface.